CMakeLists.txt

```diff
@@ -49,13 +57,6 @@ include(include/CMakeLists.txt)
 include(src/CMakeLists.txt)
 include(third-party/CMakeLists.txt)
 
-# set library version
-set_target_properties(${LRS_TARGET} PROPERTIES
-  WINDOWS_EXPORT_ALL_SYMBOLS ON
```

Comment:
This was the failing flag for the Debug/Shared build

Reply:
What was the problem? Is it something that can be worked around to still export all symbols for debug builds?

Reply:
```
Build FAILED.

       "C:\build\librealsense\ALL_BUILD.vcxproj" (default target) (1) ->
       "C:\build\librealsense\librealuvc.vcxproj" (default target) (3) ->
       (Link target) ->
         exports.def : error LNK2001: unresolved external symbol = [C:\build\librealsense\librealuvc.vcxproj]
         C:/build/librealsense/_Type_/librealuvc.lib : fatal error LNK1120: 1 unresolved externals [C:\build\librealsense\librealuvc.vcxproj]
```
The static library was not build. It could be a cmake/msvc issue.
Also I have read there are cases where msvc generates the `.dll` but not the `.lib` : I dont know it is not building.

I dont have enough knowledge on msvc to know what is the expected behavior.

Reply:
That sounds fairly interesting (how is it getting a symbol `=` ...)
Let's leave it for now.